.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "npm"
+    directory: "/client"
```

Comment:
I'm confused. We don't have a `client` directory in `content`. The `package.json` and `package-lock.json` files sit under the root.

Reply:
good catch - fixed